packages/js-core/src/index.ts

```diff
@@ -41,6 +41,11 @@ const setEmail = async (email: string): Promise<void> => {
 };
 
 const setAttribute = async (key: string, value: any): Promise<void> => {
+  if (key === "userId") {
```

Comment:
why did we decide to do this?
I think we can allow since we cleaned up everything now :-) It wouldn't have the affect the user might want to have and he will get a new attribute class named `userId`, but if they want this we maybe shouldn't stop them from it in the future.